dali/test/python/test_operator_decoders_image.py

```diff
@@ -90,26 +93,159 @@ def log(img_type, size, device, threads):
                     run_decode(data_path, batch_size, device, threads)
                     yield log, img_type, batch_size, device, threads
 
+# ToDo - check padding behavior
+@pipeline_def
+def create_decoder_slice_pipeline(data_path, device):
+    jpegs, _ = fn.readers.file(file_root = data_path,
+                               shard_id = 0,
+                               num_shards = 1,
+                               name = "Reader")
+
+    # ToDo make window random?
+    images_sliced_1 = fn.decoders.image_slice(jpegs,
+                                              [0.1, 0.1],
+                                              [0.6, 0.6],
```

Comment:
```suggestion
                                              fn.random.uniform(range=[0.05, 0.15], shape=(2,)),
                                              fn.random.uniform(range=[0.5, 0.7], shape=(2,)),
```
Something like this would work

Reply:
Done